EmbodiedSuffering_oM/Elements/MaterialImportSources.cs

```diff
@@ -25,16 +25,23 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.ComponentModel;
+using BH.oM.Base;
 
-namespace BH.oM.EmbodiedSuffering
+
+namespace BH.oM.EmbodiedSuffering.Elements
 {
-    // You can define your own Toolkit-specific types in this namespace.
+    [Description("An Embodied Suffering object used for defining the import ratios per country for a particular material, utilised in a specific country (e.g. the United States gets all of its timber from Brazil, or that it is 50% from Brazil and 50% from Vietnam.)")]
+    public class MaterialImportSources : BHoMObject
+    {
+        [Description("List of countries from which the material was imported.")]
+        public virtual Countries ExportCountries { get; set; } = Countries.Undefined;
```

Comment:
The property name and description imply a list should be used here. Is that the intention?